dot/core/block_msg_test.go

```diff
@@ -1,92 +0,0 @@
-package core
-
-import (
-	"math/big"
-	"testing"
-	"time"
-
-	"github.com/ChainSafe/gossamer/dot/core/types"
-	"github.com/ChainSafe/gossamer/dot/network"
-	"github.com/ChainSafe/gossamer/lib/common"
-	"github.com/ChainSafe/gossamer/lib/keystore"
-	"github.com/stretchr/testify/require"
-)
-
-func TestProcessBlockRequest(t *testing.T) {
```

Comment:
where did this test go?

Reply:
just moved to service_test with the other process message tests